src/Chemfiles.jl

```diff
@@ -128,7 +128,7 @@ module Chemfiles
     <http://chemfiles.readthedocs.io/en/latest/selections.html>`_ for more
     information about the selection language.
     """
-    type Selection
+    mutable struct Selection
```

Comment:
why does this need to be mutable ?

Reply:
Because a finalizer is supplied and the memory is therefore not managed by Julia. This requires the structure to be set as mutable.

Reply:
Ok, thanks!